### DISCUSSION TITLE
Found counter Strategy while saving learning Strategy

### DISCUSSION THREAD

**Discussion by Szpilman2** [UPPAALModelChecker/UPPAAL-Meta#245] [Q&A] [answered]
Dear UPPAAL Team,

Hello,

I encountered an issue while attempting to save my learning strategy in a file format. When I initiate the save process using ```saveStrategy("Evaluation/file.json", minStra)```, UPPAAL displays a message stating "Found Counter Strategy," and the strategy saves successfully. I'm uncertain if this indicates a problem with my model or if it's a bug within the UPPAAL engine.

Your guidance on this matter would be greatly appreciated.

Best regards.

**Comment by Szpilman2** [comment]
also I have attached my model here:
[fixedModel.zip](https://github.com/UPPAALModelChecker/UPPAAL-Meta/files/14009904/fixedModel.zip)

**Comment by thorulf4** [comment, marked as answer]
Hello, it seems our confirmation pop up is bugged. Atleast when i'm testing it I get the same "Found Counter Strategy" message and it still works as expected. So I think you can continue working on your model without any issues.
 I'll create a bug report on this so we can get a better confirmation message! https://github.com/UPPAALModelChecker/UPPAAL-Meta/issues/247

However loading strategies is currently broken in the beta 5 releases so I recommend working with Uppaal 5.0 for now

**Reply by Szpilman2** [reply]
Thank you so much.
